pycbc/frame/frame.py

```diff
@@ -79,6 +79,7 @@ def _read_channel(channel, stream, start, duration):
     channel_type = lalframe.FrStreamGetTimeSeriesType(channel, stream)
     read_func = _fr_type_map[channel_type][0]
     d_type = _fr_type_map[channel_type][1]
+    print(start, duration)
```

Comment:
I guess this was a debugging print, please remove before merging.

Reply:
Removed, thanks.